i2c/ssd1306_i2c/README.adoc

```diff
@@ -42,30 +39,24 @@ The driver has 3 modes of transferring the pixels in RAM to the display (provide
 
 Horizontal addressing mode has the key advantage that we can keep one single 512 byte buffer (128 columns x 4 pages and each byte fills a page's rows) and write this in one go to the RAM (column address auto increments on writes as well as reads) instead of working with 2D matrices of pixels and adding more overhead. 
 
-[NOTE]
-======
-* The SSD1306 is able to drive 128x64 displays but as our display is 128x32, only half of the COM (common) pins are connected to the display.
-* The specific display model being used is UG-2832HSWEG02
-======
-
 == Wiring information
 
 Wiring up the device requires 4 jumpers, to connect VCC (3.3v), GND, SDA and SCL and optionally a 5th jumper for the driver RESET pin. The example here uses the default I2C port 0, which is assigned to GPIO 4 (SDA) and 5 (SCL) in software. Power is supplied from the 3.3V pin from the Pico.
 
-[[oled_i2c_wiring]]
+[[ssd1306_i2c_wiring]]
 [pdfwidth=75%]
 .Wiring Diagram for oled display via I2C.
-image::oled_i2c_bb.png[]
+image::ssd1306_i2c_bb.png[]
 
 == List of Files
 
 CMakeLists.txt:: CMake file to incorporate the example into the examples build tree.
-oled_i2c.c:: The example code.
+sd1306_i2c.c:: The example code.
```

Comment:
@JamesH65 I just tried adding this renamed example into the C SDK databook (it appears in one of the appendices), and my build-scripts spotted that you've used `sd1306_i2c.c` here, when the C file is _actually_ named `ssd1306_i2c.c`.
Also, there's a `font.h` that isn't included in this files-list in the README?

Reply:
Bugger. Will fix those.

Reply:
Hmm, the missing files are a hangover from the previous incarnation - missing in that too. Not just font.h either, a number of other files, although they are just ancillaries. Do we normally list all the files in the folder?

Reply:
> Do we normally list all the files in the folder?

I think it might basically be "these are the files of note"? :shrug:  But as different examples have been provided by different people, I dunno what consistency there is.